esphome/components/ds2484/ds2484.cpp

```diff
@@ -0,0 +1,209 @@
+#include "ds2484.h"
+
+namespace esphome {
+namespace ds2484 {
+static const char *const TAG = "ds2484.onewire";
+
+void DS2484OneWireBus::setup() {
+  ESP_LOGCONFIG(TAG, "Setting up DS2484 1-wire bus...");
```

Comment:
Seems I missed this one 😅
```suggestion
  ESP_LOGCONFIG(TAG, "Running setup");
```
We are now using this standard/common message across all components as using a common message here saves space in flash.